manifests/vhost.pp

```diff
@@ -2398,12 +2404,6 @@
     content => template('apache/vhost/_serversignature.erb'),
   }
 
-  # Template uses:
```

Comment:
This template hasn't been converted. Is that an omission? For ERB templates I'd keep the "Template uses" comment in.

Reply:
yes that is an omission